.circleci/config.yml

```diff
@@ -343,100 +470,270 @@ jobs:
-                - AUTH
-                - ENTERPRISE
-                - PFS
-                - PPS1
-                - PPS2
-                - PPS3
-                - PPS4
-                - PPS5
-                - PPS6
-                - PPS_AUTH
-                - EXAMPLES
-                - INTERNAL
-  helm:
-    when:
-      and:
-        - not: << pipeline.parameters.RELEASE_PACH_CORE >>
+  # circleci:
+  #   when:
+  #     and:
+  #       - not: << pipeline.parameters.RELEASE_PACH_CORE >>
+  #   jobs:
+  #     - build-docker-images
+  #     - build-pachctl-bin:
+  #         version: $CIRCLE_SHA1
+  #         upload: true
+  #     - check-prettier
+  #     - test-envoy
+  #     - circleci:
+  #         matrix:
+  #           parameters:
+  #             bucket:
+  #               - MISC
+  #               # If you want to update the number of PPS buckets, you'll need to
+  #               # update the value of PPS_BUCKETS above
+  #               - AUTH
+  #               - ENTERPRISE
+  #               - PFS
+  #               - PPS1
+  #               - PPS2
+  #               - PPS3
+  #               - PPS4
+  #               - PPS5
+  #               - PPS6
+  #               - PPS_AUTH
+  #               - EXAMPLES
+  #               - INTERNAL
+  # helm:
+  #   when:
+  #     and:
+  #       - not: << pipeline.parameters.RELEASE_PACH_CORE >>
+  #   jobs:
+  #     - helm-tests
+  # nightly_load_tests:
+  #   triggers:
+  #     - schedule:
+  #         cron: "0 6 * * *"
+  #         filters:
+  #           branches:
+  #             only:
+  #               - master
+  #               - 2.1.x
+  #               - 2.2.x
+  #   jobs:
+  #     - nightly-load:
+  #         matrix:
+  #           parameters:
+  #             bucket:
+  #               - LOAD1
+  #               - LOAD2
+  #               - LOAD3
+  #               - LOAD4
+  #               - LOAD5
+  #               - LOAD6
+  #               - LOAD7
+  # rootless:
+  #   when:
+  #     and:
+  #       - not: << pipeline.parameters.RELEASE_PACH_CORE >>
+  #   jobs:
+  #     - rootless
+  # deploy:
+  #   when:
+  #     and:
+  #       - not: << pipeline.parameters.RELEASE_PACH_CORE >>
+  #   jobs:
+  #     - deploy
+  # pach_core_release:
+  #   when: << pipeline.parameters.RELEASE_PACH_CORE >>
+  #   jobs:
+  #     - release_job:
+  #         filters:
+  #           branches:
+  #             only:
+  #               - 2.1.x
+  #               - 2.2.x
+  # red_hat:
+  #   jobs:
+  #     - push_redhat:
+  #         filters:
+  #           branches:
+  #             ignore: /.*/
+  #           tags:
+  #             only: /^v\d+\.\d+\.\d+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$/
+  test-release:
     jobs:
+      - helm-tests:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              #only: /^v\d+\.\d+\.\d+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$/
+              only: /^\d+\.\d+\.\d+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$/
```

Comment:
We should use this `aliases` config to capture the tag regex https://github.com/pachyderm/haberdashery/blob/master/.circleci/config.yml#L3